mkdocs.yml

```diff
@@ -48,6 +48,10 @@ nav:
     - operator-manual/metrics.md
     - operator-manual/notifications.md
     - operator-manual/troubleshooting.md
+    - Server Command Reference:
```

Comment:
That is why I proposed to add it in Operator Manual rather than  in User Guide.   The operator deploys these servers are likely to know these are server commands to be run in a container.   I don't object to name the tab to `Server configuration parameters`.

``
    spec:
      containers:
      - command:
        - argocd-server
        - --staticassets
        - /shared/app
```

Reply:
Renaned.